### PR TITLE
Added Cobertura XML coverage support

### PR DESCRIPTION
**Pull request by zachkknowbe4**
Now that PHPUnit has support for Cobertura XML, I've added support so that Codeception can make use of it.

### CODE REVIEW

File: src/Codeception/Coverage/Subscriber/Printer.php [hunk 1]
```diff
@@ -146,6 +150,12 @@ protected function printCrap4j()
         $writer->process(self::$coverage, $this->absolutePath($this->options['coverage-crap4j']));
     }
 
+    protected function printCobertura()
+    {
+        $writer = new \SebastianBergmann\CodeCoverage\Report\Cobertura;
```
Comment (Naktibalda) [inline src/Codeception/Coverage/Subscriber/Printer.php:155]: Please check if this class is defined before using it and throw exception that php-code-coverage >=9.2.0 is required.

File: tests/coverage/RemoteWithCoberturaCept.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+$I = new CoverGuy($scenario);
+$I->wantTo('try generate remote cobertura xml report');
```
Comment (Naktibalda) [inline tests/coverage/RemoteWithCoberturaCept.php:3]: This test fails with PHPUnit 8 and older so it must be skipped if `\SebastianBergmann\CodeCoverage\Report\Cobertura` class does not exist.

Please check if this code works:
```php
if (!class_exists(\SebastianBergmann\CodeCoverage\Report\Cobertura::class)) {
  $scenario->skip('Cobertura report requires php-code-coverage 9.2');
}
````

**Review by Naktibalda — CHANGES_REQUESTED** [review]

**Comment by Naktibalda** [comment]
Also please document your new parameter in the docblock of Run command: https://github.com/Codeception/Codeception/blob/4.1/src/Codeception/Command/Run.php#L74-L77

It is used to generate this reference page: https://codeception.com/docs/reference/Commands#Run

**Review by Naktibalda — APPROVED** [review]
Looks good now.

**Comment by zachkknowbe4** [comment]
Awesome thank you.

**Pull request merged**